doc/sphinx/changes.rst

```diff
@@ -107,13 +107,23 @@ This release is the result of 400 merged PRs, closing ~100 issues.
 | Matthieu Sozeau for the Coq development team
 |
 
-
-Changes in 8.13+beta1
-~~~~~~~~~~~~~~~~~~~~~
+Changes in 8.13.0
+~~~~~~~~~~~~~~~~~
```

Comment:
In previous versions of the changelog, we have used a reverse chronological order only for the top-level sections (major versions) and not for the subsections (beta and patch-level versions). I'm not opposed to changing this but it would be more consistent to do it everywhere in this page.

In any case, the
```
.. contents::
   :local:
```
block was really intended for the beta subsection, so should be kept attached to it.

Reply:
done